web/web.go

```diff
@@ -990,6 +990,15 @@ func incomingWebhook(c *api.Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	channelName := parsedRequest.ChannelName
+	webhookType := parsedRequest.Type
+
+	if parsedRequest.Attachments != nil {
```

Comment:
Well I suppose we could, though 5mb is quite a lot (since attachments is only text content, which would translate to rougly 5,000,000 chars) - I'd guess an upper limit of even 1mb should be enough?

post.Props is ok :)
Incoming webhooks should always use props to specify any arbitrary data they want to save - attachments is only in there for slack compatibility and will (if set) be stored in props:

````go
	if parsedRequest.Attachments != nil {
               //...
		parsedRequest.Props["attachments"] = parsedRequest.Attachments
		webhookType = model.POST_SLACK_ATTACHMENT
	}
````
This is done because slack does not send a `type` property which would be used to decide which component to use for rendering